<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-03-23T14:24:08Z" name="Default suite" finished-at="2021-03-23T14:24:10Z" duration-ms="1537">
    <groups>
      <group name="cucumber">
        <method signature="TestRunner.feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:com.qa.customerCreation.runner.TestRunner@38c5cc4c]" name="feature" class="com.qa.customerCreation.runner.TestRunner"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2021-03-23T14:24:08Z" name="Default test" finished-at="2021-03-23T14:24:10Z" duration-ms="1537">
      <class name="com.qa.customerCreation.runner.TestRunner">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:com.qa.customerCreation.runner.TestRunner@38c5cc4c]" started-at="2021-03-23T19:54:08Z" name="setUpClass" finished-at="2021-03-23T19:54:09Z" duration-ms="345" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:com.qa.customerCreation.runner.TestRunner@38c5cc4c]" started-at="2021-03-23T19:54:09Z" name="feature" description="Runs Cucumber Feature" data-provider="features" finished-at="2021-03-23T19:54:10Z" duration-ms="1064" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Creation of Customers in Netsuite]]>
              </value>
            </param>
          </params>
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: java.lang.NullPointerException
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at com.qa.customerCreation.runner.TestRunner.feature(TestRunner.java:30)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
	at com.qa.customerCreation.ExtentReportsListeners.ExtentReportListener.testStepHandle(ExtentReportListener.java:30)
	at com.qa.customerCreation.stepDefinition.customerCreationStepDef.create_a_Customer_using_excel_data_at(customerCreationStepDef.java:45)
	at âœ½.Then Create a Customer using excel data at "C:\Users\Sravan Kumar\Desktop\Process Automation Excel Data\Customer Data.xlsx,Customer"(F:/Selenium/CustomerCreationProcessAutomation/src/main/java/com/qa/customerCreation/feature/customerCreation.feature:4)
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:com.qa.customerCreation.runner.TestRunner@38c5cc4c]" started-at="2021-03-23T19:54:10Z" name="tearDownClass" finished-at="2021-03-23T19:54:10Z" duration-ms="16" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.qa.customerCreation.runner.TestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
